package.json

```diff
@@ -26,7 +26,8 @@
     "deploy:set-yield-vault-role:local": "forge script script/deploy/SetYieldVaultRole.s.sol:SetYieldVaultRole --code-size-limit $LOCAL_CODE_SIZE_LIMIT_BYTES --rpc-url http://localhost:8545 --private-key $LOCAL_PRIVATE_KEY --sender $LOCAL_SENDER --broadcast",
     "deploy:vault:local": "forge script script/deploy/DeployVault.s.sol:DeployVault --code-size-limit $LOCAL_CODE_SIZE_LIMIT_BYTES --rpc-url http://localhost:8545 --private-key $LOCAL_PRIVATE_KEY --sender $LOCAL_SENDER --broadcast",
     "deploy:twab-delegator:local": "forge script script/deploy/DeployTwabDelegator.s.sol:DeployTwabDelegator --code-size-limit $LOCAL_CODE_SIZE_LIMIT_BYTES --rpc-url http://localhost:8545 --private-key $LOCAL_PRIVATE_KEY --sender $LOCAL_SENDER --broadcast",
-    "deploy:contracts:local": "npm run deploy:stable-token:local && npm run deploy:token:local && npm run deploy:pool:local && npm run deploy:yield-vault:local && npm run deploy:vault:local && npm run deploy:twab-delegator:local",
+    "deploy:twab-rewards:local": "forge script script/deploy/DeployTwabRewards.s.sol:DeployTwabRewards --code-size-limit $LOCAL_CODE_SIZE_LIMIT_BYTES --rpc-url http://localhost:8545 --private-key $LOCAL_PRIVATE_KEY --sender $LOCAL_SENDER --broadcast",
```

Comment:
Why do we need `--code-size-limit $LOCAL_CODE_SIZE_LIMIT_BYTES`?

Reply:
I don't know, I just followed the other scripts in the group. We can probably remove it everywhere.